orderbook/src/main.rs

```diff
@@ -120,6 +124,7 @@ async fn main() {
 
     let event_updater =
         EventUpdater::new(settlement_contract.clone(), database.clone(), sync_start);
+    let vault_updater = BalancerEventUpdater::new(vault_contract, BalancerPools::default(), None);
```

Comment:
Should probably leave instantiating + adding this component until it doesn't panic anymore.